source/guide_ackee.rst

```diff
@@ -74,15 +74,15 @@ Create a new, empty ``.env`` file in the root of the Ackee_ directory. Add the f
   ACKEE_MONGODB=mongodb://<admin>_mongoroot:<password>@localhost:27017/admin
```

Comment:
Do you think it would make sense to emphasize the lines here to be changed?

Reply:
Totally. Is it possible to highlight individual words in a code block? I only found `:emphasize-lines: X` but that--unsurprisingly--only emphasizes whole lines.

Reply:
Nope. You'd need to emphasize all line where something needs to be changed.

Reply:
Yeah, too bad. I think we can leave out the emphasize then, since every line in this block has to be changed in some way.

```
ACKEE_MONGODB=mongodb://<admin>_mongoroot:<password>@localhost:27017/admin
ACKEE_USERNAME=<ackee_username>
ACKEE_PASSWORD=<ackee_password>
ACKEE_ALLOW_ORIGIN="https://isabell.uber.space"
```

Reply:
Yes, that's why I asked :)